src/build.py

```diff
@@ -545,7 +569,7 @@ def NoSync(*args):
     Source('cmake', '', '',  # The source and git args are ignored.
            custom_sync=SyncPrebuiltCMake),
     Source('nodejs', '', '',  # The source and git args are ignored.
-           custom_sync=SyncPrebuiltNodeJS, no_windows=True),
+           custom_sync=SyncPrebuiltNodeJS, no_windows=False),
```

Comment:
Done. Eventually I hope to remove `no_windows` entirely. Although it's likely we won't want to run every test or step on every platform so we may want something more general instead.